Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:14-buster-slim
+# install base development tools
+RUN apt-get update && apt-get install -y git ruby ruby-dev build-essential rsync vim chromium parallel xvfb
```

Comment:
What is xvfb for?

Reply:
I thought that it would be needed to run puppeteer tests in a chromium browser in a x11 framebuffer.
But it uses a "headless" browser, I believe now. Will try to remove it. Same for "vim".
ruby will come from feature and be removed